patches/tModLoader/Terraria/ModLoader/DamageClass.cs

```diff
@@ -81,6 +81,13 @@ public abstract class DamageClass : ModType, ILocalizedModType
 	/// <param name="damageClass">The <see cref="DamageClass"/> you want to inherit effects from.</param>
 	/// <returns><see langword="false"/> by default - which does not let any other classes' effects trigger on this <see cref="DamageClass"/>.</returns>
 	public virtual bool GetEffectInheritance(DamageClass damageClass) => false;
+	/// <summary> 
+	/// This lets you define the classes that this <see cref="DamageClass"/> will count as (other than itself) for the purpose of prefixes<br/>
+	/// </summary>
+	/// <remarks>Return <see langword="true"/> for each <see cref="DamageClass"/> you want to inherit from</remarks>
+	/// <param name="damageClass">The <see cref="DamageClass"/> you want to inherit prefixes from.</param>
+	/// <returns><see langword="false"/> by default - which does not let any other classes' prefixes roll or remain on this <see cref="DamageClass"/>.</returns>
+	public virtual bool GetPrefixInheritance(DamageClass damageClass) => GetEffectInheritance(damageClass);
```

Comment:
Documentation needs to say `returns GetEffectInheritance by default` rather than `false`

Reply:
Should also add the note to use `GetsPrefixesFor` instead of this method when querying. And the same for `CountsAsClass` to `GetEffectInheritance`

Reply:
Added and fixed the comments